src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,109 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int INITIAL_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private static final int RESIZE_FACTOR = 2;
+    private int currentCapacity;
+    private int size;
+    private int threshold;
+    private Node<K, V>[] table;
+
+    {
+        table = new Node[INITIAL_CAPACITY];
+        currentCapacity = INITIAL_CAPACITY;
+        threshold = (int) (INITIAL_CAPACITY * LOAD_FACTOR);
+    }
```

Comment:
where should we initialize all data?

Reply:
didn't catch. in initialization block, do we?